code/modules/mob/living/simple_animal/borer/borer.dm

```diff
@@ -411,19 +418,21 @@ var/total_borer_hosts_needed = 10
 	set desc = "Become invisible to the common eye."
 
 	if(victim)
-		src << "<span class='warning'>You cannot do this whilst you are infesting a host</span>"
+		src << "<span class='warning'>You cannot do this inside a host</span>"
 
 	if(src.stat != CONSCIOUS)
```

Comment:
Also that message on 421 would read better as "You cannot do this *while you're* inside a host**.**"